docs/asyncapi-docs/src/main/scala/sttp/tapir/docs/asyncapi/AsyncAPIDocsOptions.scala

```diff
@@ -1,10 +1,12 @@
 package sttp.tapir.docs.asyncapi
 
 import sttp.tapir.Endpoint
+import sttp.tapir.SchemaType.SObjectInfo
 
 case class AsyncAPIDocsOptions(
-    subscribeOperationId: (Vector[String], Endpoint[_, _, _, _]) => String,
-    publishOperationId: (Vector[String], Endpoint[_, _, _, _]) => String
+                                subscribeOperationId: (Vector[String], Endpoint[_, _, _, _]) => String,
+                                publishOperationId: (Vector[String], Endpoint[_, _, _, _]) => String,
+                                schemaObjectInfoToNameMapper: SObjectInfo => String = SObjectInfo.defaultToNameMapper
```

Comment:
maybe let's use a simpler name - I think just `schemaName` would be sufficient, the rest is obvious from the type signature

Reply:
corrected